mozilla_django_oidc_db/config.py

```diff
@@ -34,6 +37,17 @@ def get_setting_from_config(config: OpenIDConnectConfigBase, attr: str, *args) -
        ``config._meta.get_field(some_field)``.
     """
     attr_lowercase = attr.lower()
+
+    if attr_lowercase == "oidc_op_auth_endpoint" and hasattr(
+        config.oidc_provider_config, "oidc_op_authorization_endpoint"
+    ):
+        return getattr(config.oidc_provider_config, "oidc_op_authorization_endpoint")
```

Comment:
```suggestion
        return config.oidc_provider_config.oidc_op_authorization_endpoint
```
See https://docs.astral.sh/ruff/rules/get-attr-with-constant/

why wouldn't the `config` have this attribute though? Isn't it a fixed model field?